pycm/pycm_obj.py

```diff
@@ -866,3 +866,87 @@ def to_array(self, normalized=False, one_vs_all=False, class_name=None):
             row = [table[key][i] for i in classes]
             array.append(row)
         return numpy.array(array)
+
+    def combine(self, other):
+        """
+        Return the combine of self Confusion matrix and other one.
```

Comment:
Would you please reword this sentence? My suggestion is: "For combining the confusion matrix with another one."

Reply:
Fixed in https://github.com/sepandhaghighi/pycm/pull/344/commits/b5eb3bc08c02b59bd9a452cfa786c0ca9dd412f0